<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>521</width>
                <height>261</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>MainWindow</string>
        </property>
        <property name="toolTip">
            <string>Refresh Bridge Status</string>
        </property>
        <widget class="QWidget" name="centralwidget">
            <widget class="QLabel" name="label">
                <property name="geometry">
                    <rect>
                        <x>10</x>
                        <y>50</y>
                        <width>101</width>
                        <height>31</height>
                    </rect>
                </property>
                <property name="font">
                    <font>
                        <pointsize>16</pointsize>
                    </font>
                </property>
                <property name="text">
                    <string>Light</string>
                </property>
            </widget>
            <widget class="QComboBox" name="lightsList">
                <property name="geometry">
                    <rect>
                        <x>130</x>
                        <y>50</y>
                        <width>301</width>
                        <height>31</height>
                    </rect>
                </property>
                <property name="font">
                    <font>
                        <pointsize>16</pointsize>
                    </font>
                </property>
            </widget>
            <widget class="QComboBox" name="bridgeList">
                <property name="geometry">
                    <rect>
                        <x>130</x>
                        <y>10</y>
                        <width>301</width>
                        <height>31</height>
                    </rect>
                </property>
                <property name="font">
                    <font>
                        <pointsize>16</pointsize>
                    </font>
                </property>
            </widget>
            <widget class="QLabel" name="label_2">
                <property name="geometry">
                    <rect>
                        <x>10</x>
                        <y>10</y>
                        <width>101</width>
                        <height>31</height>
                    </rect>
                </property>
                <property name="font">
                    <font>
                        <pointsize>16</pointsize>
                    </font>
                </property>
                <property name="text">
                    <string>Bridge</string>
                </property>
            </widget>
            <widget class="QToolButton" name="refreshBridges">
                <property name="geometry">
                    <rect>
                        <x>440</x>
                        <y>10</y>
                        <width>71</width>
                        <height>31</height>
                    </rect>
                </property>
                <property name="font">
                    <font>
                        <pointsize>16</pointsize>
                    </font>
                </property>
                <property name="text">
                    <string>()</string>
                </property>
                <property name="icon">
                    <iconset>
                        <normaloff>resources/icons/refresh.png</normaloff>resources/icons/refresh.png
                    </iconset>
                </property>
                <property name="iconSize">
                    <size>
                        <width>64</width>
                        <height>64</height>
                    </size>
                </property>
            </widget>
            <widget class="QLabel" name="label_3">
                <property name="geometry">
                    <rect>
                        <x>10</x>
                        <y>90</y>
                        <width>101</width>
                        <height>31</height>
                    </rect>
                </property>
                <property name="font">
                    <font>
                        <pointsize>16</pointsize>
                    </font>
                </property>
                <property name="text">
                    <string>Input File</string>
                </property>
            </widget>
            <widget class="QLineEdit" name="inputFileName">
                <property name="geometry">
                    <rect>
                        <x>130</x>
                        <y>90</y>
                        <width>301</width>
                        <height>31</height>
                    </rect>
                </property>
                <property name="font">
                    <font>
                        <pointsize>16</pointsize>
                    </font>
                </property>
            </widget>
            <widget class="QToolButton" name="chooseFile">
                <property name="geometry">
                    <rect>
                        <x>440</x>
                        <y>90</y>
                        <width>71</width>
                        <height>31</height>
                    </rect>
                </property>
                <property name="font">
                    <font>
                        <pointsize>16</pointsize>
                    </font>
                </property>
                <property name="text">
                    <string>...</string>
                </property>
                <property name="iconSize">
                    <size>
                        <width>64</width>
                        <height>64</height>
                    </size>
                </property>
            </widget>
            <widget class="QPushButton" name="startButton">
                <property name="geometry">
                    <rect>
                        <x>10</x>
                        <y>130</y>
                        <width>501</width>
                        <height>81</height>
                    </rect>
                </property>
                <property name="font">
                    <font>
                        <pointsize>18</pointsize>
                    </font>
                </property>
                <property name="text">
                    <string>Let's Dance!</string>
                </property>
                <property name="icon">
                    <iconset>
                        <normaloff>resources/icons/music.png</normaloff>resources/icons/music.png
                    </iconset>
                </property>
                <property name="iconSize">
                    <size>
                        <width>32</width>
                        <height>32</height>
                    </size>
                </property>
            </widget>
        </widget>
        <widget class="QMenuBar" name="menubar">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>521</width>
                    <height>19</height>
                </rect>
            </property>
        </widget>
        <widget class="QStatusBar" name="statusbar"/>
        <action name="actionChange_Bridge">
            <property name="text">
                <string>Change &amp;Bridge</string>
            </property>
        </action>
    </widget>
    <resources/>
    <connections/>
</ui>
